python/requirements/ml/requirements_rllib.txt

```diff
@@ -11,7 +11,7 @@ kaggle_environments==1.7.11
 # Unity3D testing
 mlagents_envs==0.27.0
 # For tests on PettingZoo's multi-agent envs.
-pettingzoo==1.11.1
+pettingzoo==1.13.1
```

Comment:
just trying to learn. do we actually publish these version requirements?
for example, if someone updates Ray version without updating their pettingzoo package, this will break their code right?

Reply:
I belive it will attempt to install the new version of pettingzoo when you update ray. Even if they cancel or avoid that automatic process, it will at least give a warning message during installation if they try to update ray without pettingzoo.